app/services/void_participant_declaration.rb

```diff
@@ -6,10 +6,10 @@ class VoidParticipantDeclaration
   def call
     declaration = ParticipantDeclaration.find(id)
```

Comment:
Shouldn't this be fetching declarations scoped for that lead_provider? Wouldn't this let one lead provider void another lead provider's declarations.